### PR TITLE
kth largest element from 2 sorted arrays

### PR DESCRIPTION
**Pull request by eshthakkar**

### CODE REVIEW

**Comment by eshthakkar** [comment]
@vkaushal21

File: closing_paran_index.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+            else:
+                last_seen_opener = st.pop()
+                if not paran_vocab[last_seen_opener] == char:
+                    return False
+
+    return len(st) == 0                
+
+
+
```
Comment (vkaushal21) [inline closing_paran_index.py]: Remove redundant white spaces.

File: closing_paran_index.py [hunk 2]
```diff
@@ -0,0 +1,95 @@
+# O(n) time and O(1) space
+def get_closing_paran_index(sentence, open_paran_index):
+    """ Return the closing paranthesis index from the input string
```
Comment (vkaushal21) [inline closing_paran_index.py]: Problem statement: 

Example:

**Review by vkaushal21 — CHANGES_REQUESTED** [review]

File: word_cloud.py [hunk 3]
```diff
@@ -0,0 +1,108 @@
+class WordCloudData(object):
```
Comment (vkaushal21) [inline word_cloud.py:1]:
```
# Problem statement 

# Best runtime and space complexity found in online solutions

# My runtime and space complexity 

# How this solutions could have been 

# Input 

# Output 

# Example input 

# Example output 
```

File: max_rides.py [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+        mid_row_index = get_middle_indices(rows)
+
+
+    # get middle indices/ index for cols
+    if cols % 2 == 0:
+        mid_col_index1 , mid_col_index2 = get_middle_indices(cols)
+    else:
+        mid_col_index = get_middle_indices(cols) 
+
+    if rows % 2 == 0 and cols % 2 == 0:
```
Comment (vkaushal21) [inline max_rides.py:30]:
```
midSet = set()
for (int i = 0; i < 2; i++) {
	for (int j = 0; j<2; j++) {
		mid.add[(rows - i)/2, (columns-j)/2]
	}
}
```

**Pull request merged**